src/pilot_ew.c

```diff
@@ -362,10 +362,10 @@ int pilot_inRangeJump( const Pilot *p, int i )
    if (hide==0.)
       return 1;
 
-   sense = EW_JUMPDETECT_DIST * p->stats.ew_jump_detect;
+   sense = EW_JUMPDETECT_DIST * p->stats.ew_jump_detect * p->stats.ew_detect;
    /* Handle hidden jumps separately, as they use a special range parameter. */
    if (jp_isFlag(jp, JP_HIDDEN))
-      sense *= p->stats.misc_hidden_jump_detect;
+      sense *= p->stats.misc_hidden_jump_detect * p->stats.ew_detect;
```

Comment:
Since it's multiplying, no need to multiply again by `p->stats.ew_detect` here, or you effectively make ew_detect affect hidden jumps twice.